config/maintainers.json

```diff
@@ -22,10 +22,10 @@
     },
     {
       "github_username": "NobbZ",
-      "show_on_website": false,
+      "show_on_website": true,
       "alumnus": false,
-      "name": null,
-      "bio": null,
+      "name": "Norbert Melzer",
+      "bio": "Found out about erlang via exercism, got hooked up and after fixing some trivial bugs, I have been added to the team,",
```

Comment:
Hello. There's an image on [the meta issue](https://github.com/exercism/meta/issues/20#issuecomment-317449929) of the current design - it's about 40words there and I think that'll be about right although the design/layout might change slightly.